Zilon.Core/Zilon.Core.Tests/Tactics/StaticObjectTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using NUnit.Framework;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Analyzer 'SonarAnalyzer.Rules.CSharp.NullPointerDereference' threw an exception of type 'SonarAnalyzer.SymbolicExecution.SymbolicExecutionException' with message 'Error processing method: AddModule_From ## Method file: StaticObjectTests.cs ## Method line: 39,8 ## Inner exception: System.NullReferenceException: Object reference not set to an instance of an object ##   at SonarAnalyzer.SymbolicExecution.CSharpExplodedGraph.VisitDefaultExpression (Microsoft.CodeAnalysis.SyntaxNode instruction, SonarAnalyzer.Sy](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5368096)